packages/neuron-ui/src/styles/theme.scss

```diff
@@ -1,47 +1,55 @@
 body {
-  --primary-color: #00C891;
-  --primary-text-color: #FFFFFF;
-  --main-background-color: #E6F3F3;
-  --secondary-background-color: #FFFFFF;
+  --primary-color: #00c891;
```

Comment:
Are these colors formatted by rules defined in the project?
If they are updated by local rules, restoring them or defining the rules in the project would be better.